utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -83,9 +83,19 @@
     public final static int DEFAULT_AUTOSCALE_POLICY_INTERVAL_TIME = 30;
     public final static int DEFAULT_AUTOSCALE_POLICY_QUIET_TIME = 5 * 60;
     private final static Random s_rand = new Random(System.currentTimeMillis());
-
-    public static long createSequenceBasedMacAddress(final long macAddress) {
-        return macAddress | 0x060000000000l | (long)s_rand.nextInt(32768) << 25 & 0x00fffe000000l;
+    private final static long prefix = 0x1e;
+
+    public static long createSequenceBasedMacAddress(final long macAddress, long globalConfig) {
```

Comment:
Is it acceptable for ``macAddress`` or ``globalConfig`` to be less than zero?  Is the maximum value permitted?

Also, please add unit test case(s) to verify the behavior of this method.